src/pages/STG_PanelAllocations.page

```diff
@@ -1,113 +1,81 @@
 <apex:page controller="STG_PanelAllocations_CTRL" docType="html-5.0">
-    <c:STG_PageHeader sectionLabel="{!$Label.stgNavDonations}" pageLabel="{!$Label.stgNavAllocations}" />
     <style>
         input.lookupInput {
-            display: inline-block;
-            height: 34px;
-            padding: 6px 12px;
-            font-size: 14px;
-            line-height: 1.428571429;
-            color: #555555;
-            background-color: #ffffff;
+            background-color: white;
```

Comment:
do we use this loookupInput css anywhere else in settings?  If so, can we create a shared settings css file to use?